### ISSUE TITLE
InvalidGraph Error onnxruntime.InferenceSession Converted from TensorFlow MobilenetV3

### ISSUE THREAD

**Issue by tnkjm** [microsoft/onnxruntime#5189]
**Describe the bug**
I could convert my fine tuned TensorFlow model ( MobileNet v3 ) to onnx without Error.
However, When I run
sess = onnxruntime.InferenceSession("MY_ONNX_MODEL")
I got InvalidGraph Error.

**System information**
- OS Windows10
- ONNX Runtime installed from binary
- ONNX Runtime version:1.4.0
- Python version:3.7.9
- onnx:1.7.0
- tensorflow:1.14.0
- tf2onnx:1.6.3

**To Reproduce**
1. Download file from follwing URL
http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v3_small_coco_2019_08_14.tar.gz

2. Train with donloaded File.

3. Convert TensorFlow model to ONNX.

   if you convert  saved model to ONNX

    python -m tf2onnx.convert --opset 10 --fold_const --saved-model PATH/TO/saved_model --output 
    PATH/TO/model.onnx

    if you convert  frozen model to ONNX

    python -m tf2onnx.convert --graphdef PATH/TO/frozen_inference_graph.pb ^
    --output PATH/TO/model.frozen.onnx ^
    --fold_const --opset 10 ^
    --inputs image_tensor:0 ^
    --outputs num_detections:0,detection_boxes:0,detection_scores:0,detection_classes:0

4. Run sess = onnxruntime.InferenceSession("PATH/TO/model.onnx")

**Expected behavior**
run without error

**Screenshots**
![InvalidGraph](https://user-images.githubusercontent.com/34013580/93296039-5dd7e900-f829-11ea-81a6-5211bcf36950.PNG)

**Comment by RandySheriffH** [comment]
@TomWildenhain-Microsoft, mind to take a look?

**Comment by TomWildenhain-Microsoft** [comment]
I think your output names might be incorrect.  I converted with:
`python -m tf2onnx.convert --graphdef "PATH/TO/frozen_inference_graph.pb" --output result.onnx --opset 10 --inputs normalized_input_image_tensor:0 --outputs raw_outputs/box_encodings:0,raw_outputs/class_predictions:0`
And the inference session was created properly.

**Comment by tnkjm** [comment]
Thank you for looking into it.

I could n't convert with 
`python -m tf2onnx.convert --graphdef "PATH/TO/frozen_inference_graph.pb" --output result.onnx --opset 10 --inputs normalized_input_image_tensor:0 --outputs raw_outputs/box_encodings:0,raw_outputs/class_predictions:0`


I fine-tune mobilenet v3.
When I run 
`saved_model_cli show --dir PATH/TO/MY/FINE-TUNE/saved_model --tag_set serve  --signature_def serving_default`
it shows 

The given SavedModel SignatureDef contains the following input(s):
  inputs['inputs'] tensor_info:
      dtype: DT_UINT8
      shape: (-1, -1, -1, 3)
      name: image_tensor:0
The given SavedModel SignatureDef contains the following output(s):
  outputs['detection_boxes'] tensor_info:
      dtype: DT_FLOAT
      shape: (-1, 10, 4)
      name: detection_boxes:0
  outputs['detection_classes'] tensor_info:
      dtype: DT_FLOAT
      shape: (-1, 10)
      name: detection_classes:0
  outputs['detection_multiclass_scores'] tensor_info:
      dtype: DT_FLOAT
      shape: (-1, 10, 14)
      name: detection_multiclass_scores:0
  outputs['detection_scores'] tensor_info:
      dtype: DT_FLOAT
      shape: (-1, 10)
      name: detection_scores:0
  outputs['num_detections'] tensor_info:
      dtype: DT_FLOAT
      shape: (-1)
      name: num_detections:0
  outputs['raw_detection_boxes'] tensor_info:
      dtype: DT_FLOAT
      shape: (-1, -1, 4)
      name: raw_detection_boxes:0
  outputs['raw_detection_scores'] tensor_info:
      dtype: DT_FLOAT
      shape: (-1, -1, 14)
      name: raw_detection_scores:0
Method name is: tensorflow/serving/predict

**Comment by TomWildenhain-Microsoft** [comment]
The model you uploaded only has one protobuf file (frozen_inference_graph.pb).  Can you upload the fine-tuned saved model you are converting?

**Comment by tnkjm** [comment]
Sorry, this is saved model.
[saved_model.pb.zip](https://github.com/microsoft/onnxruntime/files/5248718/saved_model.pb.zip)

**Comment by TomWildenhain-Microsoft** [comment]
Thanks.  I was able to reproduce the error:
`INVALID_GRAPH : This is an invalid model. Error in Node:generic_loop_Loop__94 : Node (Pad__1203) has input size 2 not in range [min=1, max=1]`
Looking into it now.

**Comment by TomWildenhain-Microsoft** [comment]
It looks like there is a bug in converting NonMaxSuppressionV4 for opset 10.  If you use opset 11 it should work.  I'm working on fixing the bug now.

**Comment by TomWildenhain-Microsoft** [comment]
https://github.com/onnx/tensorflow-onnx/pull/1109

**Comment by tnkjm** [comment]
Thank you. 
opset 11 worked without InvalidGraph Error.

**Comment by TomWildenhain-Microsoft** [comment]
Awesome.  The PR merged now so it should work on opset 10 if you need.  @RandySheriffH this issue can be closed.
